### ISSUE TITLE
Does batch normalization increase the accuracy?

### ISSUE THREAD

**Issue by twmht** [Joker316701882/Additive-Margin-Softmax#14]
Hi,

I've tried batch normalization with resnet20 in caffe. But I found out the test accuracy drops.

Do you compare the accuracy with or w/o Batch normalization?

**Comment by Joker316701882** [comment]
@twmht 
Sorry, not yet.

**Comment by twmht** [comment]
As you suggested. I am trying ADAM now. Any feedback will be reported here.

**Comment by Joker316701882** [comment]
@twmht 
Hey, I updated this repo yesterday, some bugs are fixed and the conclusion before may have some problem. I found that Mom is better in face recognition. And with resface20(bn), lfw reaches 99.5%， I think batch norm actually increase the performance compared to original paper. Further more, in deeper models, batch norm is a standard configuration. So I think just use it.

**Comment by twmht** [comment]
@Joker316701882 

Hi, So you just append batch norm in each convolution of resnet20, and you don't append the batch norm after the fully connection (which is used as the embedding)?
